manifests/unit_file.pp

```diff
@@ -81,6 +84,7 @@
   Optional[Boolean]                        $hasrestart = undef,
   Boolean                                  $hasstatus = true,
   Boolean                                  $selinux_ignore_defaults = false,
+  Hash                                     $service_parameters = {},
```

Comment:
Didn't check the whitespace, but how about this?
```suggestion
  Hash[String[1], Any]                                     $service_parameters = {},
```

Reply:
ah yes, implemented it.